src/features/island/bumpkin/components/LevelUp.tsx

```diff
@@ -0,0 +1,162 @@
+  20: "Level 20, you're the cream of the crop!",
+  21: "Twenty-one and harvesting like a pro!",
+  22: "Congrats on level 22! Your farm is getting plowed with success.",
+  23: "Level 23, your skills are really starting to bloom!",
+  24: "You're really blossoming at level 24!",
+  25: "Quarter-century mark! You're making hay while the sun shines.",
+  26: "Congrats on level 26! You're really milking this farming thing.",
+  27: "Level 27, you're really starting to stand out in the field!",
+  28: "You're really raising the bar at level 28!",
+  29: "Congrats on level 29! You're really crop-tivating some serious skills.",
+  30: "Level 30, you're a true farmer now!",
+  31: "Thirty-one and still growing strong!",
+  32: "Congrats on level 32! Your farm is in full bloom.",
+  33: "Level 33, your farming skills are really taking off!",
+  34: "You're really sprouting at level 34!",
+  35: "Level 35, you're the tractor-trailer of farming!",
+  36: "Congrats on level 36! Your farm is really starting to harvest some success.",
+  37: "Level 37, your skills are really starting to crop up!",
+  38: "You're really planting the seeds of success at level 38!",
+  39: "Congrats on level 39! Your farm is really starting to mature.",
+  40: "Level 40, you're a harvesting hero!",
+  41: "Forty-one and still growing strong!",
+  42: "Congrats on level 42! Your farm is starting to rake in the rewards.",
+  43: "Level 43, you're really cultivating some serious skills.",
+  44: " You're really harvesting success at level 44!",
+  45: " Level 45, you're a true master of the harvest!",
+  46: "Congrats on level 46! Your farming skills are really starting to bear fruit.",
+  47: "Level 47, you're really growing into a farming legend.",
+  48: "You're really thriving at level 48!",
+  49: "Congrats on level 49! You're really starting to reap the rewards of your hard work.",
+  50: "Halfway to 100! You're a true farming pro now.",
+  51: "Fifty-one and still going strong!",
+  52: "Congrats on level 52! Your farm is a true work of art.",
+  53: "Level 53, your skills are really starting to take root.",
+  54: "You're really harvesting happiness at level 54!",
+  55: "Level 55, you're a true farming force to be reckoned with.",
+  56: "Congrats on level 56! Your farm is really starting to blossom.",
+  57: "Level 57, you're really starting to cultivate some serious skills.",
+  58: "You're really sowing the seeds of success at level 58!",
+  59: "Congrats on level 59! Your farm is the cream of the crop.",
+  60: " Level 60, you're a true farming superstar!",
+};
+
+const IMAGE_WIDTH = 160;
+
+interface Props {
+  level: number;
+  onClose: () => void;
+  wearables: Equipped;
+}
+export const LevelUp: React.FC<Props> = ({ level, onClose, wearables }) => {
+  const shareMessage = `Just reached level ${level} in Sunflower Land! So proud of my progress in this game. 🌻🚀 \n\n https://www.sunflower-land.com \n\n #SunflowerLand #LevelUp`;
```

Comment:
Perhaps this could be the players referal link

Reply:
Not a bad idea 👌 

For now, I will let the player themselves decide if they want to add that one